masonry/src/widget/checkbox.rs

```diff
@@ -74,24 +74,19 @@ impl Widget for Checkbox {
                 }
             }
             PointerEvent::PointerUp(_, _) => {
-                if ctx.is_active() && !ctx.is_disabled() {
-                    if ctx.is_hot() {
-                        self.checked = !self.checked;
-                        ctx.submit_action(Action::CheckboxChecked(self.checked));
-                        trace!("Checkbox {:?} released", ctx.widget_id());
-                    }
-                    ctx.request_paint();
+                if ctx.is_active() && ctx.is_hot() && !ctx.is_disabled() {
```

Comment:
No, it's a drive-by fix.

Reply:
I really don't like drive-by fixes much, but recognize I may well be in the minority ...

Why submit 1 PR when you can submit 5? :)

But this won't block things ...

Reply:
I think it depends on the complexity (i.e. is it worth it to notify a lot of people because of some mini-code-aesthetic issue), but I also usually think it's better to isolate PRs